multinode_integration_tests/src/masq_mock_node.rs

```diff
@@ -193,12 +201,27 @@ impl MASQMockNode {
             framer,
             chain: TEST_DEFAULT_MULTINODE_CHAIN,
         };
-        MASQMockNode {
+        ConfigurableMASQMockNode {
             control_stream,
-            guts,
+            configurable_guts,
         }
     }
 
+    pub fn absorb_configuration(&mut self, node_record: &NodeRecord) {
+        // Copy attributes from the NodeRecord into the MASQNode.
+        self.configurable_guts.earning_wallet = node_record.earning_wallet();
+        self.configurable_guts.rate_pack = *node_record.rate_pack();
+    }
+
+    pub fn finish_with_guts_pointer(self) -> MASQMockNode {
```

Comment:
Maybe should be an implementation of Into<MASQMockNode>? Or From<...> in the other direction?